src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -362,60 +363,131 @@ export class EducationWizard extends React.Component {
+                        onClick={() =>
+                          recordEvent({
+                            event: 'edu-navigation',
+                            'edu-action': 'stem-scholarship',
+                          })
+                        }
+                      >
+                        Edith Nourse Rogers STEM Scholarship
+                      </a>
+                      , you must meet all the requirements below. You:
+                    </strong>
+                    <ul className="wizard-edith-nourse-content">
+                      <li>
+                        Are using or recently used Post-9/11 GI Bill or Fry
+                        Scholarship benefits
+                      </li>
+                      <li>
+                        Have used all your education benefits or are within 6
+                        months of doing so.{' '}
+                        <a
+                          className="checkBenefitsLink"
+                          href="../gi-bill/post-9-11/ch-33-benefit/"
+                          onClick={() =>
+                            recordEvent({
+                              event: 'edu-navigation',
+                              'edu-action': 'check-remaining-benefits',
+                            })
+                          }
+                        >
+                          Check remaining benefits
+                        </a>
+                      </li>
+                      <li>
+                        Are enrolled in an undergraduate degree program for
+                        science, technology, engineering or math (STEM),{' '}
+                        <strong>or</strong> have already earned a STEM degree
+                        and are pursuing a teaching certification.{' '}
+                        <a
+                          href="https://benefits.va.gov/gibill/docs/fgib/STEM_Program_List.pdf"
+                          onClick={() =>
+                            recordEvent({
+                              event: 'edu-navigation',
+                              'edu-action': 'see-approved-stem-programs',
+                            })
+                          }
+                        >
+                          See approved STEM programs
+                        </a>
+                      </li>
+                    </ul>
+                  </div>
+                ) : (
+                  <div>
+                    <strong>
+                      To be eligible for the{' '}
+                      <a
+                        href="https://benefits.va.gov/gibill/fgib/stem.asp"
+                        onClick={() =>
+                          recordEvent({
+                            event: 'edu-navigation',
+                            'edu-action': 'stem-scholarship',
+                          })
+                        }
+                      >
+                        Edith Nourse Rogers STEM Scholarship
+                      </a>
```

Comment:
May want to add the `rel` attribute -

```suggestion
                      <a
                        href="https://benefits.va.gov/gibill/fgib/stem.asp"
                        rel="noopener noreferrer"
                        onClick={() =>
                          recordEvent({
                            event: 'edu-navigation',
                            'edu-action': 'stem-scholarship',
                          })
                        }
                      >
                        Edith Nourse Rogers STEM Scholarship
                      </a>
```